evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -823,3 +824,14 @@ def test_explain_predictions_stacked_ensemble(problem_type, dummy_stacked_ensemb
 
     with pytest.raises(ValueError, match="Cannot explain predictions for a stacked ensemble pipeline"):
         explain_predictions_best_worst(pipeline, X, y)
+
+
+@pytest.mark.parametrize("estimator", [e for e in _all_estimators() if ('Classifier' in e.name and not any(s in e.name for s in ["Baseline", "Cat", "Elastic", "KN", "Ensemble"]))])
+def test_explain_predictions_oversampler(estimator, fraud_100):
+    pytest.importorskip('imblearn.over_sampling', reason='Skipping test because imbalanced-learn not installed')
+    X, y = fraud_100
+    pipeline = BinaryClassificationPipeline(component_graph=["Imputer", "One Hot Encoder", "DateTime Featurization Component", "SMOTENC Oversampler", estimator])
+    pipeline.fit(X, y)
+    report = explain_predictions(pipeline, X, y, indices_to_explain=[0], output_format="dataframe", top_k_features=4)
+    assert report['feature_names'].isnull().sum() == 0
+    assert report['feature_values'].isnull().sum() == 0
```

Comment:
Yep this seems reasonable to me. We have other tests I believe which check that the output of prediction explanations is what we expect it to be. This test is just checking that something came out which matches the form we expect. 👍